website/kiosks/please-check-in.kiosk

```diff
@@ -15,32 +15,37 @@ $(function () { setInterval(function() { location.reload(true); }, 30000); });
 </script>
 <style>
 body { overflow: hidden; }
-.kiosk_heading {
-  border-bottom: 2px solid black;
-}
 .wanted_container {
-  column-width: 250px;
   position: absolute;
-  top: 100px;
-  bottom: 0;
-}
-.wanted_container ul {
-    -moz-column-count: 5;
```

Comment:
These -moz- and -webkit- attribute prefixes were added in #227, and I'd like to keep them for wider compatibility.